src/test/resources/saReport.txt

```diff
@@ -0,0 +1,31 @@
++ Test security analysis report
+  + AC security analysis on network 'sim1'
+    + Network CC0 SC0
+      + Post loading processing
+         Network has 4 buses and 4 branches
+         Network balance: active generation=607.0 MW, active load=600.0 MW, reactive generation=0.0 MVar, reactive load=200.0 MVar
+      + Pre-contingency simulation
+         AC load flow complete with NR status 'CONVERGED'
+        + Outer loop Distributed slack on generation
+           Iteration 0: failed to distribute slack bus active power mismatch, -1.4404045651219555 MW remains
+        + Outer loop Reactive limits
```

Comment:
Maybe here another possible improvement: should we discard empty child reporters?